partner_contact_in_several_companies/models/ir_actions.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class IRActionsWindow(models.Model):
+    _inherit = 'ir.actions.act_window'
+
+    @api.multi
+    def read(self, fields=None, context=None, load='_classic_read'):
+        actions = super(IRActionsWindow, self).read(fields=fields, load=load)
+        for action in actions:
+            if action.get('res_model', '') == 'res.partner':
+                # By default, only show standalone contact
+                action_context = action.get('context', '{}') or '{}'
+                if 'search_show_all_positions' not in action_context:
+                    action['context'] = action_context.replace(
+                        '{',
+                        ("{'search_show_all_positions': "
+                         "{'is_set': True, 'set_value': False},"),
+                        1)
+        return actions
```

Comment:
It seems to me the preceding lines can not work, or am I missing something? Tried in ipython:
```
(buildoutenv3) openeyedev@laptop17 ~ $ ipython3
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.3.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: action_context = {}                                                                                                                                                  

In [2]: action_context.replace('{', ("{'search_show_all_positions': {'is_set': True, 'set_value': False},"), 1)                                                              
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-7d214fbc2633> in <module>
----> 1 action_context.replace('{', ("{'search_show_all_positions': {'is_set': True, 'set_value': False},"), 1)

AttributeError: 'dict' object has no attribute 'replace'
```